spec/tictactoe_clojure/board_spec.clj

```diff
@@ -1,7 +1,43 @@
 (ns tictactoe-clojure.board-spec
-  (:require [speclj.core :refer :all]
-            [tictactoe-clojure.board :refer :all]))
+	(:require [speclj.core :refer :all]
+		[tictactoe-clojure.board :refer :all]))
+
+(describe "tictactoe-clojure.board"
+	(describe "new-board"
+ 		(it "returns a vector with a range from 0 to 8"
+			(should= new-board [0 1 2 3 4 5 6 7 8] )))
```

Comment:
`[0 1 2 3 4 5 6 7 8]` seems to be repeated a few times here. We could pull this out and refer to it. Similar comment about the "X" and "O"'s